Gemfile

```diff
@@ -1,19 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.7.1'
+gem 'rails', '~> 5.1'
 
 gem 'unicorn', '4.9.0'
 
 gem 'logstasher', '0.4.8'
-gem 'rack_strip_client_ip', '0.0.1'
-gem 'actionpack-page_caching', '1.0.2'
+gem 'rack_strip_client_ip', '0.0.2'
+gem 'actionpack-page_caching', '1.1.0'
 
 gem 'uglifier', ">= 1.3.0"
 gem 'sass-rails', "5.0.6"
-gem 'airbrake', '~> 4.3.1'
```

Comment:
In most apps we've been using our own fork of airbrake 4 to get rails 5 compatibility and avoid losing some functionality on our errbit install (see: https://docs.publishing.service.gov.uk/manual/upgrade-rails.html#airbrake)